src/components/NewUiAddressDetailExplorer.js

```diff
@@ -0,0 +1,644 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import PaginationURL from '../utils/pagination';
+import hathorLib from '@hathor/wallet-lib';
+import WebSocketHandler from '../WebSocketHandler';
+import { TX_COUNT, TOKEN_COUNT } from '../constants';
+import { isEqual, find } from 'lodash';
+import metadataApi from '../api/metadataApi';
+import addressApi from '../api/addressApi';
+import txApi from '../api/txApi';
+import ErrorMessageWithIcon from '../components/error/ErrorMessageWithIcon';
+import NewUiAddressHistory from './NewUiAddressHistory';
+import NewUiAddressSummary from './NewUiAddressSummary';
+import Spinner from './Spinner';
+
+class NewUiAddressDetailExplorer extends React.Component {
```

Comment:
question: In #317 this component was rewritten using functional components, which would allow using the hooks needed. Please use that component to reuse the complex logic and add only the new interface elements.

Reply:
Removed to use the refactored functional component. Please review.